src/main/java/seedu/address/logic/commands/deletecommands/DeleteInternshipCommand.java

```diff
@@ -22,6 +24,7 @@
  */
 public class DeleteInternshipCommand extends DeleteCommand {
     public static final String MESSAGE_SUCCESS = "Internship of index %1$s in %2$s successfully deleted.";
+    public static final String DISPLAY_MESSAGE_SUCCESS = "Deleted the internship at index %1$s from this company: ";
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(